main.py

```diff
@@ -1,12 +1,172 @@
+    while choice not in valid_choices:
+        choice = input("CHOOSE A NUMBER--> ")
+        if choice not in list_options():
+            print("YOU MUST SELECT A VALID CHOICE")
+    return choice
+       
+
+#runs entire program
+def main(play=True):
+    video_store_management = VideoStoreManagement(url=BACKUP_URL)
+    while play==True:
+        logo()
+        options = list_options()#prints user options 
+        choice = user_choice(options)#users choice from menu
+        
+        if choice == '1':#create customer
+            print("SAWEEET.. LETS DO THIS..LETS GET THIS NEW CUSTOMER CREATED!")
+            name=input("Customer first name --> ")
+            postal_code=input(f"{name}'s postal code --> ")
+            phone=input(f"{name}'s phone number --> ")
+            video_store_management.create_customer(name,postal_code,phone)
+            print(f"WHEWW..{name}'s profile was successfully created!")
+        
+        if choice == '2':#get customer
+            id=input("ENTER THE CUSTOMERS ID --> ")
+            search_by_id = video_store_management.get_customer(id=id)
+            print(search_by_id)
+        
+        if choice == '3':#update customer
+            id = input("ENTER CUSTOMERS ID YOU WANT TO UPDATE --> ")
+            customer_to_update = video_store_management.get_customer(id=id)
+            update = input("LETS GET THIS CUSTOMER UP TO DATE! FOR NAME UPDATE PRESS N,  PHONE UPDATE PRESS P, POSTAL CODE PRESS Z  ")
+            if update == 'N':
+                name = input("ENTER THE NEW NAME HERE --> ")
+                customer_to_update["name"] = name
+            elif update == 'P':
+                phone = input("NEW PHONE NUMBER GOES HERE --> ")
+                customer_to_update["phone"] = name
+            elif update == 'Z':
+                postal_code =input("NEW POSTAL CODE GOES HERE -->")
+            query_params = {
+                "id":customer_to_update["id"],
+                "postal_code": customer_to_update["postal_code"],
+                "phone": customer_to_update["phone"],
+                "name": customer_to_update["name"]
+                }
+            response = requests.put(url=BACKUP_URL+f"/customers/{id}",json=query_params)
+            print(response.json())
+            return response.json()
+            
+        
+        if choice =='4':#delete customer
+            id = input("ENTER CUSTOMERS ID YOU WANT TO DELETE --> ")
+            video_store_management.delete_customer(id=id)
+
+        if choice == '5':#view customers
+            for c in video_store_management.list_customers():
+                print(c)
+        
+        if choice =='6':#create video
+            title=input("YO... SO WHATS THIS NEW VIDS TITLE --> ")
+            release_date=input(f" ENTER THE RELEASE DATE FOR {title} YYYY-MM-DD--> ")
+            video_store_management.create_video(title=title,release_date=release_date)
+            print(f"WHEWW..{title} was successfully created!")
```

Comment:
This won't actually be true in an error case. See my comment on the create_video method for more details.